scripts/sync-db-push.sh

```
@@ -0,0 +1,77 @@
+echo "==> 服务器端备份现有 db..."
+ssh "$REMOTE_HOST" "cp ${REMOTE_PATH} ${REMOTE_PATH}.bak.${TIMESTAMP}"
+echo "✓ 备份完成: ${REMOTE_PATH}.bak.${TIMESTAMP}"
+
+# 2. 上传
+echo "==> 上传中..."
+scp -q "$LOCAL_PATH" "${REMOTE_HOST}:${REMOTE_PATH}"
+echo "✓ 上传完成"
+
+# 3. 重启服务
+echo "==> 重启服务 ${REMOTE_SERVICE}..."
+ssh "$REMOTE_HOST" "systemctl restart ${REMOTE_SERVICE} && sleep 2 && systemctl is-active ${REMOTE_SERVICE}" || {
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)